ai-platform-unified/notebooks/official/automl/tabular-model-training-and-prediction.ipynb

```diff
@@ -0,0 +1,719 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.    if delete_bucket and "BUCKET_NAME" in globals():
</pre><p>Can you make a slight change?</p><p> </p><p>if os.getenv("IS_TESTING"):</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/361/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Reply:
Nvm, I'll do it for you.

Reply:
I'll do it across multiple notebooks.